core/src/main/javascript/reportviewer/reportviewer-prompt.js

```diff
@@ -715,7 +719,6 @@ define([
 
             //Timezone is not found in a value, check the hint
             if (timezoneHint) {
```

Comment:
Why is the below replace not needed anymore (e.g from `\+xxx` to `+xxx`)?

Reply:
The Timezone hint is calculated on the server side and we removed the backslash in the java code. The check was only necessary if we were initially calculating it on the server side as \+xxx.
The original code check here was also never working, as the regex was not a proper format and always left it as \+xxx, then when it went to compare dates, it would always return "true" for an invalid value and never update. By fixing it initially and then removing it, we uncovered other issues that required other javascript fixes (as seen in this file too).

Reply:
Yes, it looks like it was a missing a slash to make it work `/\\\+/`. I missed that on the first read.
Anyway, I didn't see any server-side code removal related with timezone hints and thus my doubt.
I assume the removal was made in the context of another issue.
Thanks!